lib/trento/application/process_managers/deregistration_process_manager.ex

```diff
@@ -0,0 +1,62 @@
+defmodule Trento.DeregistrationProcessManager do
+  @moduledoc """
+    DeregistrationProcessManager is a Commanded ProcessManager, it's the responsible
+    for the deregistration procedure for both hosts and clusters.
+
+    This represent a transaction to ensure that the procedure of deregistering hosts and cluster
```

Comment:
`This represent"s"`.
And the same for `hosts and clusters`, this excludes sap systems message